CMakeLists.txt

```diff
@@ -36,6 +36,12 @@ add_custom_target(luatest
   COMMENT "Run regression tests"
 )
 
+add_custom_target(luatest-no-coverage
```

Comment:
@DifferentialOrange @slavakirichenko Shouldn't we have make targets like these?

- luatest
- luatest-coverage (or something like this)

I think it is more logical and convenient.

Reply:
It sounds better, but let's not break backward compatibility yet (for example, we may rename the target to `luatest-coverage` and add an alias if CMake can do so)

Reply:
Actually, I am ok with the current approach, it was just a question.

Reply:
Yeah, let's at least use `luatest` here.

Reply:
I mean `luatest` instead of `luatest-no-coverage`

Reply:
@DifferentialOrange but we alerady have one with coverage at CMakeLists.txr add_custom_target(**luatest**

Reply:
Ok, sorry. Let's leave it as is for now